<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0" colspan="7">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="4" column="5">
     <widget class="KeyCombo" name="altCombo7"/>
    </item>
    <item row="2" column="0">
     <widget class="KeyCombo" name="comboLeft"/>
    </item>
    <item row="5" column="5">
     <widget class="KeyCombo" name="combo3"/>
    </item>
    <item row="0" column="1">
     <widget class="KeyCombo" name="comboUp"/>
    </item>
    <item row="5" column="6">
     <widget class="KeyCombo" name="combo4"/>
    </item>
    <item row="3" column="6">
     <widget class="KeyCombo" name="combo8"/>
    </item>
    <item row="2" column="2">
     <widget class="KeyCombo" name="comboRight"/>
    </item>
    <item row="3" column="1">
     <widget class="KeyCombo" name="comboDown"/>
    </item>
    <item row="3" column="4">
     <widget class="KeyCombo" name="combo6"/>
    </item>
    <item row="3" column="3">
     <widget class="KeyCombo" name="combo5"/>
    </item>
    <item row="3" column="5">
     <widget class="KeyCombo" name="combo7"/>
    </item>
    <item row="5" column="3">
     <widget class="KeyCombo" name="combo1"/>
    </item>
    <item row="5" column="0">
     <widget class="KeyCombo" name="combo9"/>
    </item>
    <item row="5" column="4">
     <widget class="KeyCombo" name="combo2"/>
    </item>
    <item row="4" column="3">
     <widget class="KeyCombo" name="altCombo5"/>
    </item>
    <item row="4" column="4">
     <widget class="KeyCombo" name="altCombo6"/>
    </item>
    <item row="4" column="6">
     <widget class="KeyCombo" name="altCombo8"/>
    </item>
    <item row="6" column="3">
     <widget class="KeyCombo" name="altCombo1"/>
    </item>
    <item row="6" column="4">
     <widget class="KeyCombo" name="altCombo2"/>
    </item>
    <item row="6" column="5">
     <widget class="KeyCombo" name="altCombo3"/>
    </item>
    <item row="6" column="6">
     <widget class="KeyCombo" name="altCombo4"/>
    </item>
    <item row="6" column="0">
     <widget class="KeyCombo" name="altCombo9"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KeyCombo</class>
   <extends>QComboBox</extends>
   <header>keycombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
